README.md

```diff
@@ -1,3 +1,8 @@
+# Status
+
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-shakas/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-shakas?branch=master)
```

Comment:
You've only changed the second line, what about the first? It's still develop

Reply:
🤔 I see. Correcting right away.

Reply:
Hi @FawazFarid. I have made the changes.